src/include/distributed/multi_planner_utils.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * multi_planner_utils.h
+ *
+ * This file contains functions helper functions for planning
+ * queries with colocated tables and subqueries.
+ *
+ * Copyright (c) 2017-2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#ifndef MULTI_PLANNER_UTILS_H
+#define MULTI_PLANNER_UTILS_H
+
+#include "distributed/multi_planner.h"
+
+
+extern bool SafeToPushDownSubquery(PlannerRestrictionContext *plannerRestrictionContext);
```

Comment:
This is only used inside multi_router_planner.c , declared in multi_planner_utils.h, and defined in relation_restriction_equivalance.c

It looks weird.